locales/en/translation.json

```diff
@@ -21,6 +23,8 @@
   "Capturing Selected Changes…": "Capturing Selected Changes…",
   "Change Developer": "Change Developer",
   "Change Reviewer": "Change Reviewer",
+  "Changes requested": "Changes requested",
```

Comment:
Maybe this translation can be removed since we are using title case.

Reply:
It's used in the org card, where we're not currently using title case.